KappaAnalysis/src/Producers/ElectronCorrectionsProducer.cc

```diff
@@ -84,5 +106,11 @@ void ElectronCorrectionsProducer::Produce(KappaEvent const& event, KappaProduct&
 void ElectronCorrectionsProducer::AdditionalCorrections(KElectron* electron, KappaEvent const& event,
                                                         KappaProduct& product, KappaSettings const& settings) const
 {
+	//size_t iter = 0;
```

Comment:
If `iter` is no longer needed, you can delete this line.

Reply:
Done

Reply:
Nice, thanks.